src/style.css

```
@@ -1 +1,21 @@
-/* add styles here */
+body {
+  margin: 0;
+  display: block;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url("star.822d561e.svg");
+  background-position: 50%;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:nth-child(-n+1), .stars--2 .stars__star:nth-child(-n+2), .stars--3 .stars__star:nth-child(-n+3), .stars--4 .stars__star:nth-child(-n+4), .stars--5 .stars__star:nth-child(-n+5) {
+  background-image: url("star-active.edda4c43.svg");
+}
```

```suggestion
.stars--1 .stars__star:nth-child(-n+1),
.stars--2 .stars__star:nth-child(-n+2),
.stars--3 .stars__star:nth-child(-n+3),
.stars--4 .stars__star:nth-child(-n+4),
.stars--5 .stars__star:nth-child(-n+5) {
  background-image: url("star-active.edda4c43.svg");
}
```

Don't forget about code formatting to make your code readable